script/config.js

```diff
@@ -56,9 +56,10 @@ function getChannel (version) {
   return 'stable'
 }
 
-function getChannelDisplayName (channel) {
-  if (channel === 'stable' || channel === 'dev') return null
-  return process.env.ATOM_CHANNEL_DISPLAY_NAME || channel.charAt(0).toUpperCase() + channel.slice(1)
+function getAppName (channel) {
+  return channel === 'stable'
+    ? 'Atom'
+    : `Atom ${process.env.ATOM_CHANNEL_DISPLAY_NAME || channel.charAt(0).toUpperCase() + channel.slice(1)}`
```

Comment:
Custom display name support, eh? Time to build "Atom 3000" :sparkles:

Reply:
Haha, yeah, just in case we want some one-off release branches :)